spec/integration/_dev/benchmarks/scenario.spec.yml

```diff
@@ -0,0 +1,30 @@
+spec:
+  type: object
+  additionalProperties: false
+  required:
+    - version
+    - data_stream
+    - description
+    - corpora
+  properties:
+    version:
+      $ref: "../integration/manifest.spec.yml#/definitions/version"
+    data_stream:
+      description: The data-stream to benchmark.
+      type: string
+    description:
+      description: A description of the benchmark scenario.
+      type: string
+    corpora:
+      description: Corpora settings.
+      type: object
+      additionalProperties: false
+      required:
+        - input_service
+      properties:
+        input_service:
+          description: The service (found in ./deploy/docker) that will send corpora logs to the elastic agents.
+          type: string
+    warmup_time_period:
+      description: A time period in seconds for warmup of the benchmark. All response data captured during warmup will not show up in the measurement results.
+      type: integer
```

Comment:
This type does not matches the values given in the test packages (strings):
```yaml
warmup_time_period: 1m
```

If we set string as a type, probably it would be better some kind of regex to allow just duration format (1m, 1s, 1h... ?).

Or should it be updated the test package?